core/app/helpers/mno_enterprise/image_helper.rb

```diff
@@ -0,0 +1,22 @@
+module MnoEnterprise
+  module ImageHelper
+    # Helper method to easily access and select the images
+    def select_image
+      path = "/app/assets/images/"
+
+      main_logo_whitebg = "#{path}mno_enterprise/main-logo-whitebg.png"
+      return image(main_logo_whitebg) if image(main_logo_whitebg)
+
+      main_logo = "#{path}mno_enterprise/main-logo.png"
+      return image(main_logo) if image(main_logo)
+
+      return "#{MnoEnterprise::Engine.root}#{main_logo}"
+    end
+
+    def image(logo)
```

Comment:
Same here. No comments and a generic name.
The logic of building the path is split between the 2 methods.
We want to give priority to the local image and default to the one from mnoe.

I think I'd rewrite the module like the following, I find it a lot easier to follow.
```ruby
# Add some comments here!
def main_logo_whitebg
  app_image(""mno_enterprise/main-logo-whitebg.png"") || main_logo
end

# Add some comments here!
def main_logo
  logo_path = "mno_enterprise/main-logo.png"
  app_image_path(logo_path) || engine_image_path(logo_path)
end

# Add some comments (eg: get an image from the app etc)
def app_image_path(path)
  app_path = Rails.root.join("/app/assets/images/", path)
  app_path if File.exists?(app_path)
end

def engine_image_path(path)
  engine_path = MnoEnterprise::Engine.root.join("/app/assets/images/", path)
end
```